src/main/Validator.js

```diff
@@ -61,6 +61,29 @@ const configDataSchemaV1 = Joi.object({
   spellCheckerLocale: Joi.string().regex(/^[a-z]{2}-[A-Z]{2}$/).default('en-US'),
 });
 
+const configDataSchemaV2 = Joi.object({
+  version: Joi.number().min(2).default(1),
```

Comment:
```suggestion
  version: Joi.number().min(2).default(2),
```
also, I'm not really sure if min should be `1`, as I don't know if the validator is running before or after the upgrade

Reply:
min should be 2 for the v2 config, it should fail on that and go through the upgrade process.